cmake/GTClangOptions.cmake

```diff
@@ -27,6 +27,9 @@ option(GTCLANG_ASSERTS "Enable asserts" ${BUILD_IS_NOT_RELEASE})
 option(GTCLANG_USE_CCACHE "Use compile cache (ccache)" ON)
 option(GTCLANG_BUILD_EXAMPLES "Build examples (requires boost, gridtools and OpenMP/CUDA)" ON)
 option(GTCLANG_BUILD_EXAMPLES_WITH_GPU "Use the gpu (CUDA) backend when compiling gridtools" OFF)
+if(GTCLANG_BUILD_EXAMPLES_WITH_GPU) 
+  set(GPU_DEVICE "P100" CACHE TYPE STRING)
```

Comment:
why do we hard set it to the P100 here and later check for multiple ones? Can I manually override this? If so, where?

Reply:
yes, this is a CACHE variable, which means that it can, should be set by the user running cmake 
-DGPU_DEVICE=K80 or using ccmake